proc_scripts/plotting.py

```diff
@@ -14,6 +14,7 @@ def expand_limits(thisrange,s):
             retval[j]*sgn[j] > full_range[j]*sgn[j]
             else
             retval[j] for j in range(2))
+
 def draw_limits(thisrange,s):
     full_range = s.getaxis('t2')[r_[0,-1]]
     dw = diff(s.getaxis('t2')[:2]).item()
```

Comment:
docstring

Reply:
you thumbs-upped but didn't add a docstring!

Reply:
should all be there now! :)